src/engraving/dom/tie.cpp

```diff
@@ -355,30 +355,66 @@ void Tie::calculateDirection()
     }
 }
 
-void Tie::calculateIsInside()
+void Tie::calculateIsInside(bool isUp)
```

Comment:
One last thing: it doesn't make sense for this method to take `isUp` as a parameter. You are inside the Tie class here, so you can use the up() method directly to access that piece of data (instead of accessing it from the caller by passing `item->up()`). That way, the function signature can stay as it was

Reply:
Thanks for the comprehensive feedback!